### ISSUE TITLE
Usage in github action will create wrong links

### ISSUE THREAD

**Issue by sajadghawami** [technikhil314/auto-badger#12]
hey,

thank you for the amazing package!

Locally everything is working fine, but when i try to use the package in a github action, some of the links are broken.
The repository name is '@codestra/authentication-manager'.

The result will be as follows:

```
[![Build Status](https://img.shields.io/github/workflow/status//codestra/Push%20on%20master?style=flat-square&color=%23007a1f)](https://github.com//codestra/actions)

....

[![stargazers](https://img.shields.io/github/stars//codestra?style=social)](https://github.com//codestra/stargazers)
[![number of forks](https://img.shields.io/github/forks//codestra?style=social)](https://github.com//codestra/fork)
```

As you can see, there are `//` at some point in the url.

Some of the images actually do work, but some are broken. This is the whole Readme: [Generated Readme](https://raw.githubusercontent.com/codestra/authentication-manager/master/README.md)

Any clue how to fix that?

**Comment by technikhil314** [comment]
@sajadghawami Yeah I will have to investigate this further. Can you help me with the node version you are using and the `remote "origin"` section from your `.git/config` file? It should look something like following. You will have to log it in your action to help me dig more into it. 
```
[remote "origin"]
	url = git@github.com-personal:technikhil314/auto-badger.git
	fetch = +refs/heads/*:refs/remotes/origin/*
```

Thank you for using. I am glad you find it useful.
